cant build in ubuntu 20.04

i getting an error while building. is there any possibility to install simple as a plugin without these complex building procedure ?
this is the error i got.
```
CMake Error at CMakeLists.txt:6 (find_package):
  By not providing "FindLibObs.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "LibObs", but
  CMake did not find one.

  Could not find a package configuration file provided by "LibObs" with any
  of the following names:

    LibObsConfig.cmake
    libobs-config.cmake

  Add the installation prefix of "LibObs" to CMAKE_PREFIX_PATH or set
  "LibObs_DIR" to a directory containing one of the above files.  If "LibObs"
  provides a separate development package or SDK, be sure it has been
  installed.


-- Configuring incomplete, errors occurred!

```

--- a/README.md
+++ b/README.md
@@ -36,6 +36,12 @@ issue #17][vcam#17].
 sudo apt install qtbase5-dev
 ```
 
+- Install LibObs
+
+```
+sudo apt install libobs-dev
+```
+
 - Get obs-studio source code
 
 ```

